src/main/kotlin/net/atos/zac/mail/MailService.kt

```diff
@@ -82,11 +86,37 @@ class MailService @Inject constructor(
         private const val MAIL_BIJLAGE = "Bijlage"
         private const val MAIL_ONDERWERP = "Onderwerp"
         private const val MAIL_BERICHT = "Bericht"
+
+        private const val MAIL_SMTP_USER = "mail.smtp.user"
+        private const val MAIL_SMTP_AUTH = "mail.smtp.auth"
     }
 
     val gemeenteMailAdres
         get() = MailAdres(configuratieService.readGemeenteMail(), configuratieService.readGemeenteNaam())
 
+    @Synchronized
+    private fun setupPasswordAuthentication() {
+        if (authenticationSetupDone) return
+
+        val userName = System.getenv("SMTP_USERNAME")
```

Comment:
Is it not possible to use `@ConfigProperty` like we do for all other env vars (even if it is optional now)? That would make it more consistent.